src/lib/style-formatting/style-serializer.ts

```diff
@@ -322,7 +321,7 @@ function encodeObject({ location, key, value, style, explode, escape, isAllowedR
  */
 function encodePrimitive({ location, key, value, style, escape, isAllowedReserved = false }: StylizerConfig) {
   const valueEncoder = (str: string) =>
-    module.exports.encodeDisallowedCharacters(str, {
```

Comment:
Is there an eslint rule to flag these that we should add in to the esm eslint config of ours? @kanadgupta

Reply:
@erunion definitely — I can PR that!

Reply:
https://github.com/readmeio/standards/pull/763